src/mmw/js/src/data_catalog/views.js

```diff
@@ -1,24 +1,93 @@
 "use strict";
 
-var Marionette = require('../../shim/backbone.marionette'),
+var _ = require('underscore'),
+    $ = require('jquery'),
+    Backbone = require('../../shim/backbone'),
     dataCatalogWindowTmpl = require('./templates/window.html'),
-    searchbarTmpl = require('./templates/searchbar.html');
+    searchResultsTmpl = require('./templates/searchresults.html');
 
-var DataCatalogWindow = Marionette.LayoutView.extend({
-    template: dataCatalogWindowTmpl,
+var ENTER = 13;
 
-    regions: {
-        'searchBar': '#data-catalog-searchbar-region',
+var dom = {
+    searchBox: '.data-catalog-search-input',
+    searchResults: '.data-catalog-results',
+    catalogButton: '.data-catalog-sources > button',
+    resource: '.data-catalog-results > .resource',
+    expandLink: '[data-action="expand"]'
+};
+
+var DataCatalogWindow = Backbone.View.extend({
```

Comment:
It was simpler and faster to implement this without Marionette. Now that some of the details are a little more fleshed out, I'll see if using Marionette is viable.